crossing-repair.lic

```diff
@@ -87,11 +87,30 @@ class CrossingRepair
   end
 
   def setup
+    arg_definitions = [
+      [
+        { name: 'town', regex: /Crossing|River.*|Theren.*|Shard|Hib.*|Ratha/i, optional: true, description: 'Town to sell in' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
+    
+    town = args.town.capitalize
+    if town.start_with?("Hib")
+      town = "Hibarnhvidar"
+    elsif town.start_with?("River")
+      town = "Riverhaven"
+    elsif town.start_with?("Theren")
+      town = "Therenborough"
+    end
+    settings = get_settings
+
+    settings['hometown'] = town if town
```

Comment:
💡 I'm not a fan of mutating `settings`, I would have expected something more like `@hometown = town || settings.hometown`

Reply:
I'm not a huge fan of this either.  I think the common-money functions need to be updated to take more specific parameters.  I'll work on getting that refactored, but it'll probably be a pretty big PR because a lot of other scripts touch that code